torchao/float8/float8_linear_utils.py

```diff
@@ -101,6 +102,7 @@ def convert_to_float8_training(
     Returns:
      nn.Module: The modified module with swapped linear layers.
     """
+    torch._C._log_api_usage_once("torchao.quantization.convert_to_float8_training")
```

Comment:
@vkuzo I put this under the `torchao.quantization` namespace based on some past discussions on code reorg that hasn't happened yet. Do you prefer this namespace compared to `torchao.float8`, which is the current location?

Reply:
yeah and also we can start tracking earlier, not sure when the migration will happen

Reply:
IMO better to match existing code structure, and if a refactor happens we'll deal with the logging change.  I say this because there is no timeline or owner for actually migrating `torchao.float8` at the moment.